node/index.js

```diff
@@ -11,16 +11,35 @@ const speechGoogle = require('./speech_google');
 const speechOpenAI = require('./speech_openai');
 const { state } = require('./state');
 
+// see versions here: https://github.com/wppconnect-team/wa-version/tree/main/html
+const wwebVersion = '2.2412.54';
+
+const puppeteerOptions = {
+        headless: true,
+        args: ['--no-sandbox', '--disable-setuid-sandbox']
+};
+
+if (env.chromiumPath) {
```

Comment:
Is this optional or should we fail gracefully?
What if chromiumPath is null?

Reply:
seems like the highlight moved, I wanted to highlight the if-condition on line 22+

Reply:
It's optional: if it's there the executablePath gets set, otherwise it behaves like before and lets puppeteer look wherever it's looking.

I made this change b/c on alpine for some reason puppeteer did not find chromium so I need to pass the binary path in (see `runbun.sh`).